2.cellprofiler_processing/cputils.py

```diff
@@ -7,28 +7,27 @@
 import pathlib
 from pathlib import Path
 
-def rename_sqlite_file(sqlite_file_path: pathlib.Path, plate: str):
+def rename_sqlite_file(sqlite_dir_path: pathlib.Path, plate: str):
     """Rename the .sqlite file to be {plate}.sqlite as to differentiate between the files
 
     Args:
-        sqlite_file_path (pathlib.Path): path to CellProfiler_output directory
+        sqlite_dir_path (pathlib.Path): path to CellProfiler_output directory
         plate (str): name of the plate that was run through CellProfiler
     """
-    for sqlite_files in sqlite_file_path.iterdir():
-        # the CellProfiler pipeline hardcodes the .sqlite file name, so all files outputted are called "CFReT.sqlite"
-        if "CFReT.sqlite" in str(sqlite_files):
-            print(sqlite_files.name)
-            # change the file name to include the plate name
-            new_file_name = str(sqlite_files).replace(
-                sqlite_files.name, f"{plate}.sqlite"
-            )
-            # change the file name in the directory
-            Path(sqlite_files).rename(Path(new_file_name))
-            print("The file is renamed!")
-        else:
-            # if there is no file named "CFReT.sqlite" then the file name as been changed/corrected
-            print("The sqlite file has already been renamed!")
+    try:
+        # CellProfiler requires a name to be set in to pipeline, so regardless of plate, all sqlite files are outputed as "CFReT.sqlite"
+        sqlite_file_path = pathlib.Path(f"{sqlite_dir_path}/CFReT.sqlite")
+        
+        new_file_name = str(sqlite_file_path).replace(sqlite_file_path.name, f"{plate}.sqlite")
 
+        # change the file name in the directory
+        Path(sqlite_file_path).rename(Path(new_file_name))
```

Comment:
Consider changing these to `pathlib.Path` to remove unnecessary import and be consistent (or change `pathlib.Path` statements).

Reply:
Huh, I didn't even realize this! I have changed this and removed the import. Thank you!